modules/core/src/lib/attribute.js

```diff
@@ -327,6 +332,25 @@ export default class Attribute extends BaseAttribute {
     return false;
   }
 
+  hasAlias(name) {
```

Comment:
This feels risky and very loaders-specific. It assumes a certain mapping between loaders attributes and layer attributes.